mix.exs

```diff
@@ -39,4 +41,17 @@ defmodule Lob.Mixfile do
       {:uuid, "~> 1.1", only: :test}
     ]
   end
+
+  defp description do
+    "Lob Elixir Library."
+  end
+
+  defp package do
+    [
+      maintainers: ["Lob"],
+      files: ["lib/**/*.ex", "mix*", "*.md"],
```

Comment:
According the the documentation, this "defaults to standard project directories, so you usually don't need to set this property." However, I was seeing that it included `.DS_Store` in the package. I searched around and there doesn't seem to be a great way to exclude files so this is what I came up with.

Reply:
Seems reasonable. Here is what Ecto does: https://github.com/elixir-ecto/ecto/blob/master/mix.exs#L70-L72